docs/source/advanced/ipu.rst

```diff
@@ -0,0 +1,233 @@
+.. _ipu:
+
+IPU support
+===========
+
+.. note::
+    IPU Support is experimental and a work in progress (see :ref:`known-limitations`). If you run into an problems, please leave an issue.
+
+Lightning supports `Graphcore Information Processing Units (IPUs) <https://www.graphcore.ai/products/ipu>`_, processors built for Artificial Intelligence and Machine Learning.
+
+IPU Terminology
+---------------
+
+IPUs consist of many individual cores, allowing parallelization across computation. Due to the high bandwidth speed between cores,
```

Comment:
Would be cool to add a reference to IPU Arch https://www.graphcore.ai/products/ipu

Reply:
Nvm, you added it in the section above.